src/platform/silabs/efr32/BUILD.gn

```diff
@@ -69,7 +69,7 @@ static_library("efr32") {
   if (chip_enable_ble_rs911x) {
     sources += [
       "../rs911x/BLEManagerImpl.cpp",
-      "../rs911x/rsi_ble_config.h",
```

Comment:
This could be affecting rs9116/917  ncp mode

Reply:
done @chirag-silabs